src/library/Popover/Popover.js

```diff
@@ -122,7 +122,6 @@ const Root = createStyledComponent(
   },
   {
     displayName: 'Popover',
-    includeStyleReset: true,
```

Comment:
Truncate uses Tooltip, which uses Popover. Placing Truncate in a Button (when Popover included the style reset) was causing lots of issues — mostly related to fontSize & lineHeight.

Reply:
And because the main visual portion of Popover is PopoverContent, which does have the reset, and not the root Popover, there was no impact whatsoever from removing the reset from the root in any other context (e.g. when Popover/Tooltip/Dropdown is wrapped around a Button, Select, etc...).